src/decisionengine_modules/tests/etc/decisionengine/config.d/AWS_Calculations_with_source_proxy.conf

```diff
@@ -1,4 +1,5 @@
-{ "sources"  : {
+{ "channel_name": "test"
```

Comment:
I think a comma is missing at the end of the line

Reply:
Yes. Fixed.